test/file.ts

```diff
@@ -3205,6 +3206,39 @@ describe('File', () => {
     });
   });
 
+  describe('isPublic', () => {
+    const sandbox = sinon.createSandbox();
+
+    afterEach(() => sandbox.restore());
+
+    it('should execute callback with `true` in response', done => {
+      sandbox.stub(gaxios, 'request').resolves();
+      file.isPublic((err: gaxios.GaxiosError, resp: boolean) => {
+        assert.ifError(err);
+        assert.strictEqual(resp, true);
+        done();
+      });
+    });
+
+    it('should execute callback with `false` in response', done => {
+      sandbox.stub(gaxios, 'request').rejects({code: '403'});
+      file.isPublic((err: gaxios.GaxiosError, resp: boolean) => {
+        assert.ifError(err);
+        assert.strictEqual(resp, false);
+        done();
+      });
+    });
+
+    it('should propagate different error to user', done => {
```

Comment:
Changing the message to `should propagate non-403 errors to user` might help others understand how this works.

Reply:
Sure, done